parser.y

```diff
@@ -6339,6 +6341,12 @@ AdminStmt:
 			ShowSlow: $4.(*ast.ShowSlow),
 		}
 	}
+|	"ADMIN" "RELOAD" "EXPR_PUSHDOWN_BLACKLIST"
```

Comment:
should we also add a `"ADMIN" "SHOW" "EXPR_PUSHDOWN_BLACKLIST"` syntax support?

Reply:
I think that `select * from mysql.expr_pushdown_blacklist` may be enough.

Reply:
OK. Anyway, We can add it later if necessary.